### ISSUE TITLE
Enhance output products to include 1-sigma(rms, error) component outputs

### ISSUE THREAD

**Issue by karllark** [spacetelescope/jwst#5894]
Currently the output exposures/integrations and mosaics have output images in surface brightness, errors (aka 1-sigma aka rms) in surface brightness, and the 3 component error terms as variances (errors^2) in (surface brightness)^2.  It would be useful and support more end use cases and communities if the pipeline also output the 3 component error terms (read noise, photon noise, and flat field noise) as errors (aka 1-sigma aka rms).  Having the total error and the 3 component errors presented in the same units (surface brightness) in the output files would also be more intuitive.

This would require more disk space, but as long as this is not a major limitation it would remove one step in user analysis.

**Comment by jdavies-st** [comment]
@stscijgbot-jp

**Comment by melanieclarke** [comment]
@drlaw1558 - is there any interest in storing error components as 1-sigma errors in addition to or instead of variances?

**Comment by drlaw1558** [comment]
@melanieclarke No; given that we're starting to deal with issues arising from large data volumes from the output imaging products I don't think we should add new extensions that effectively duplicate existing information.  Likewise, ERR may be more intuitive than VAR, but I think the confusion that would be caused by switching our convention would outweigh any gains here.  I'm going to close this.
